international-labour-organization.csl

```diff
@@ -158,49 +169,62 @@
             </group>
           </group>
         </else-if>
-        <else-if type="chapter paper-conference" match="any">
-          <text macro="title" prefix=" " suffix="."/>
+        <else-if type="chapter" match="any">
           <group prefix=" " delimiter=" ">
-            <text term="in" text-case="capitalize-first" suffix=":"/>
+            <text variable="title" font-style="normal" suffix=","/>
+            <text term="in" text-case="lowercase" font-variant="normal"/>
             <text macro="editor"/>
             <text variable="container-title" font-style="italic" suffix="."/>
             <text variable="collection-title" suffix="."/>
+            <text variable="scale"/>
             <text macro="edition"/>
             <group suffix="." delimiter=", ">
-              <text macro="publisher"/>
+              <text macro="publisher" prefix=" "/>
               <text macro="pages"/>
             </group>
           </group>
         </else-if>
-        <else-if type="thesis">
-          <group prefix=" " delimiter=". " suffix=".">
-            <text macro="title"/>
-            <text macro="edition"/>
-            <text variable="genre"/>
-            <text macro="publisher"/>
-          </group>
-        </else-if>
-        <else-if type="webpage">
-          <group prefix=" " suffix=".">
-            <text macro="title" suffix=" [Online]."/>
-            <text macro="edition"/>
-            <text prefix=" " macro="access" suffix="."/>
-          </group>
-        </else-if>
+        <else-if type="thesis"/>
         <else-if type="article-journal">
           <text macro="title" prefix=" &quot;" suffix="&quot;,"/>
           <group prefix=" " delimiter=" ">
-            <text term="in" suffix=":"/>
+            <text term="in" text-case="lowercase" font-variant="normal"/>
             <text macro="editor"/>
             <text variable="container-title" font-style="italic" suffix=","/>
             <text macro="edition"/>
+            <text macro="publisher" prefix=" "/>
             <group suffix="." delimiter=", ">
               <text macro="publisher"/>
-              <text variable="volume"/>
-              <text macro="pages"/>
             </group>
+            <text term="volume" form="short" text-case="capitalize-first"/>
```

Comment:
I'm thinking this is what you want further up, too. But you'll want to wrap these two into a group. Otherwise the volume term is getting printed even if there is no actual volume, i.e. you'd get `Vol. pp. 123-345`